evm_loader/program/src/account_storage/apply.rs

```diff
@@ -76,7 +78,10 @@ impl<'a> ProgramAccountStorage<'a> {
         for action in &actions {
             let address = match action {
                 Action::NeonTransfer { target, .. } => target,
-                Action::EvmSetCode { address, .. } => address,
+                Action::NeonWithdraw { source, .. } => source,
```

Comment:
`Action::NeonWithdraw` probably is not needed here.
This action could only happen when `source` account already exists